### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.NullPointerException

### ISSUE THREAD

**Issue by jingyanjingyan** [microsoft/azure-tools-for-java#3814]
Build: dev 1316 19.2
System: windows, mac, linux
Note: it will pops dialog ask to open in ASE with bundle build.

Repro Steps:
Link a cluster, and open the storage account
Result:
Error

IntelliJ build version: 2019.2.4 IC-192.7142.36
OS: Windows 10
JDK: JetBrains s.r.o 11.0.4
Plugin version: 3.30.0
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,0,1384x811,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
java.lang.NullPointerException
	at java.desktop/javax.swing.ImageIcon.<init>(ImageIcon.java:217)
	at com.microsoft.azuretools.ijidea.ui.WarningMessageForm.$$$setupUI$$$(WarningMessageForm.java)
	at com.microsoft.azuretools.ijidea.ui.WarningMessageForm.<init>(WarningMessageForm.java:48)
	at com.microsoft.azure.hdinsight.common.HDInsightHelperImpl$6$1.<init>(HDInsightHelperImpl.java:254)
	at com.microsoft.azure.hdinsight.common.HDInsightHelperImpl$6.run(HDInsightHelperImpl.java:254)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:433)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

**Comment by wezhang** [comment]
@t-rufang , looks hit the issue again

**Comment by t-rufang** [comment]
Will take a look at this issue

**Comment by jingyanjingyan** [comment]
In build 1323 19.2, sign in a reader role account, expand a reader role cluster, click the jobs node and storage account node will get the NPE errors.

**Comment by jingyanjingyan** [comment]
Fix in build azure-toolkit-for-intellij-2019.2.4.develop.11088651.12-13-2019.zip (19.2&bundle)
